stdlib/asyncio/queues.pyi

```diff
@@ -10,11 +10,16 @@ if sys.version_info >= (3, 10):
 else:
     _LoopBoundMixin = object
 
-__all__ = ("Queue", "PriorityQueue", "LifoQueue", "QueueFull", "QueueEmpty")
-
 class QueueEmpty(Exception): ...
 class QueueFull(Exception): ...
 
+if sys.version_info >= (3, 13):
+    class QueueShutDown(Exception): ...
+    __all__ = ("Queue", "PriorityQueue", "LifoQueue", "QueueFull", "QueueEmpty", "QueueShutDown")
```

Comment:
`__all__` should stay at the top of the file (below the imports), even if it means repeating the version_info branch.

Reply:
Addressed: https://github.com/python/typeshed/pull/12327/commits/bee3ed25dd11701b855c0f314936097f01b2869c